taxonomies/food/ingredients.txt

```diff
@@ -60565,10 +60575,11 @@ nl: groene spliterwten
 
 # popular in Latvia
 < en:pea
-en: grey pea, grey peas
+en: grey pea, grey peas, gray pea
```

Comment:
It might make sense (additionally to or instead of this) to add a `synonyms:en: grey, gray` line at the top of the file.